From 296e2a0fb5e527a784f81effd89f6ad095a95b83 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Wed, 16 May 2018 08:52:26 +0800
Subject: [PATCH] clk: rockchip: rk3368: mark the aclk_dmac_bus as critical clk

crypto and dmac share the same noc clk,
so mark the aclk_dmac_bus as critical clk.

Change-Id: Ib0b70bbed3fdefeab7b6f2b5f88350a416e66787
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3368.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk3368.c b/drivers/clk/rockchip/clk-rk3368.c
index 39ec324e2fd3..ad96b4203209 100644
--- a/drivers/clk/rockchip/clk-rk3368.c
+++ b/drivers/clk/rockchip/clk-rk3368.c
@@ -891,6 +891,7 @@ static const char *const rk3368_critical_clocks[] __initconst = {
 	"pclk_ddrphy",
 	"pclk_ddrupctl",
 	"pmu_hclk_otg0",
+	"aclk_dmac_bus",
 };
 
 static void __init rk3368_clk_init(struct device_node *np)
-- 
2.35.3

